specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-10-01-preview/workspaceRP.json

```diff
@@ -7112,6 +7302,10 @@
           "description": "Retention time in days after workspace get soft deleted.",
           "type": "integer"
         },
+        "systemDatastoresAuthMode": {
```

Comment:
>systemDatastoresAuthMode [](http://example.com/codeflow?start=9&length=24)

being a string what the values looks like? should be an enum?

Reply:
this is already in the property for PUT, we just add it in the PATCH supported property list, let me see if I can change both place into a enum.

Reply:
looks good, not considered as breaking change. please review